usr/src/uts/armv8/os/fakebop.c

```diff
@@ -239,22 +293,49 @@ void bop_init(struct xboot_info *xbp)
 	bootops->bsys_getprop = do_bsys_getprop;
 	bootops->bsys_nextprop = do_bsys_nextprop;
 	bootops->bsys_printf = bop_printf;
+
+#if XXXARM
+	/* Set up the shadow fb for framebuffer console */
+	boot_fb_shadow_init(bops);
+#endif
+
+	/*
+	 * Start building the boot properties from the command line
+	 */
+	DBG_MSG("Initializing boot properties:\n");
+	build_boot_properties(xbp);
+
+#if XXXARM
```

Comment:
Is there a reason we can't do this now?

Reply:
This is the start of a stream of work to sort out how we do the bsvc UART and introduce the text console, which will be backed by either simplefb or the UEFI framebuffer as appropriate for the platform.

With that background, `find_boot_prop` lives in `boot_console.c`, so when I reintroduce that I'll enable these blocks.